src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -61,6 +61,7 @@ export class EditDataGridPanel extends GridParentComponent {
 	private enableEditing = true;
 	// Current selected cell state
 	private currentCell: { row: number, column: number, isEditable: boolean, isDirty: boolean };
+	private justClickedCell: { row: number, column: number };
```

Comment:
I'd suggest `lastClickedCell` - "just" implies it was recent, but this logic doesn't care about that at all. You could click and then wait a couple hours and click again and it'd still behave the same.